Makefile

```diff
@@ -104,6 +104,7 @@ build: ## Runs the Python Build.
 install: build ## Runs the build target and then installs via setup.py
 	# Debian/Ubuntu requires an additional parameter in setup.py
 	@source distro_build_configs.sh; \
+	git config --global --add safe.directory /code; \
```

Comment:
Does this need to change the global git config?

Reply:
Nope that was a copy & paste error by me. Sorry for that.